test/unit/core/entities/ds_data_class_test.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:hmi_core/hmi_core.dart';
+
+void main() {
+  group('DsDataClass', () {
```

Comment:
add test to verify that class has all necessary names with proper values:
  requestAll('requestAll'),
  requestList('requestList'),
  requestTime('requestTime'),
  requestAlarms('requestAlarms'),
  requestPath('requestPath'),
  syncTime('syncTime'),
  commonCmd('commonCmd'),
  commonData('commonData');

Reply:
Done